README.md

```diff
@@ -17,12 +17,16 @@ See [our website](https://flow-project.github.io/) for more information on the a
 - [Installation instructions](http://flow.readthedocs.io/en/latest/flow_setup.html)
 - [Tutorials](https://github.com/flow-project/flow/tree/master/tutorials)
 
+# Technical questions
```

Comment:
I recommend we move this into the Getting involved section (see my suggestions to the PR)

Reply:
I think both ways are good. 

I believe having a separate section for technical questions would be useful since if people are desperately looking for help, seeing a whole section dedicated to this may make them feel better. (also asking a question is really logically not getting involved, as opposed to reporting bugs, improvements, etc.)

But either way works for me :)

Reply:
Okay, I'm convinced actually. LGTM then, going to merge.